ambry-server/src/integration-test/java/com.github.ambry.server/RouterServerTestFramework.java

```diff
@@ -564,6 +653,21 @@ void checkDeleted() throws Exception {
       }
     }
 
+    /**
+     * Check if router got expected RouterErrorCode.
+     * @throws Exception
```

Comment:
please add java doc for `routerErrorCode`.

Reply:
added.